plugin/vscode/performance-writer.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2020  Matthew Glazar
+// See end of file for extended copyright information.
+
+const fs = require('fs');
+const path = require("path");
+const process = require("process");
+
+class PerformanceWriter {
+    
+    constructor (extensionContext) {
+        this._performanceTxtPath = path.join(process.cwd(), "test_performance.txt");
+        this._dataAttributes = {
+            columns:'timestamp,file-ref,file-name-or-duration,event-id'
+        };
+        this._fileName;
+        this._fileIndex;
```

Comment:
What are these doing? Looks like dead code. Did you mean to initialize these to `null`?
```suggestion
        this._fileName = null;
        this._fileIndex = null;
```

Reply:
Addressed in [this commit](https://github.com/lifeinData/quick-lint-js/4f4bdd46c6cdbe22e9b84ed2d69f9ed4d372fe5a)

Reply:
resolved in latest commit